BudgetTracker.Data.EntityFramework/Models/TransactionModel.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BudgetTracker.Data.EntityFramework.Models
+{
+    public class TransactionModel
+    {
+        public Guid Id { get; set; }
+        public string VendorName { get; set; }
+        public string Description { get; set; }
+        public decimal Amount { get; set; }
+        public DateTime DateOfTransaction { get; set; }
+        public string CheckNumber { get; set; }
+        [MaxLength(200)]
```

Comment:
This probably could be much bigger. The description could have a MaxLength but this should be much larger IMO